src/main/java/com/sekwah/advancedportals/bukkit/portals/Portal.java

```diff
@@ -146,11 +145,11 @@ public static void loadPortals() {
         return blockTypes;
     }
 
-    public static String create(Location pos1, Location pos2, String name, String destination, Set<Material> triggerBlocks, PortalArg... extraData) {
-        return create(pos1, pos2, name, destination, triggerBlocks, null, extraData);
+    public static String create(Location pos1, Location pos2, String name, String destination, Set<Material> triggerBlocks, int cooldown, PortalArg... extraData) {
+        return create(pos1, pos2, name, destination, triggerBlocks, null, cooldown, extraData);
     }
 
-    public static String create(Location pos1, Location pos2, String name, String destination, Set<Material> triggerBlocks, String serverName, PortalArg... portalArgs) {
+    public static String create(Location pos1, Location pos2, String name, String destination, Set<Material> triggerBlocks, String serverName, int cooldown, PortalArg... portalArgs) {
```

Comment:
Cooldown should be using the in place option of extraData. It is designed for tags such as this and is the better solution. In the current code all tags are actually stored the same way as apposed to hard coded like this.